doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
```

Comment:
might want to add this file to the `CMakeLists.txt` so it can be translated to manpage. and also need to add it to a package in `debian/control` and `ceph.spec.in`.